changelog/21.0/21.0.0/summary.md

```diff
@@ -218,3 +219,22 @@ work automatically during the [`MoveTables`](https://vitess.io/docs/reference/vr
 [`--remove-sharded-auto-increment` boolean flag](https://vitess.io/docs/20.0/reference/programs/vtctldclient/vtctldclient_movetables/vtctldclient_movetables_create/) and you should begin using the new
 [`--sharded-auto-increment-handling` flag](https://vitess.io/docs/21.0/reference/programs/vtctldclient/vtctldclient_movetables/vtctldclient_movetables_create/) instead. Please see the new
 [`MoveTables` Auto Increment Handling](https://vitess.io/docs/21.0/reference/vreplication/movetables/#auto-increment-handling) documentation for additional details.
+
```

Comment:
We generally keep the summary short and expect users to look at the vitess.io docs for the details.

Reply:
@rohit-nayak-ps makes sense, updated!